pages/_app.tsx

```diff
@@ -10,7 +11,7 @@ export default function App({ Component, pageProps }) {
 
```

Comment:
typescript infers `theme` to be `string` but this is only partially correct as it is partially incorrect.
`theme` cannot be any string! it can only be `"bright"` or `"dark"`